### PR TITLE
[ch27970] sentry-5922: Admin: agreement __str__ issue

### PR DESCRIPTION
**Pull request by emaciupe**
fix for sentry issue #ETOOLS-PROD-EH when deleting an agreement
https://excubo.unicef.io/sentry/etools-prod/issues/5922/

### CODE REVIEW

File: src/etools/applications/reports/models.py [hunk 1]
```diff
@@ -836,9 +836,12 @@ class ReportingRequirement(TimeStampedModel):
     class Meta:
         ordering = ("-end_date", )
 
+    def get_report_type_display(self):
```
[the remark this answers is not in the document]
Comment (emaciupe) [reply src/etools/applications/reports/models.py]: @domdinicola thanks a lot, it's done

**Comment by robertavram** [comment]
Since this error was a good error to have :| as it was triggering only when a user tried to delete an agreement that had PDs from Admin, we should include in this PR a way to trigger an error or make a superuser unable to delete from the admin an agreement that has interventions associated with it. an error text could be "Please delete all interventions associated with this Agreement before deleting the agreement"

**Comment by emaciupe** [comment]
> Since this error was a good error to have :| as it was triggering only when a user tried to delete an agreement that had PDs from Admin, we should include in this PR a way to trigger an error or make a superuser unable to delete from the admin an agreement that has interventions associated with it. an error text could be "Please delete all interventions associated with this Agreement before deleting the agreement"
 
The above requested changes in the admin are now included in this PR.

**Comment by robertavram** [comment]
[ch27970]

**Pull request merged**